packages/office-ui-fabric-react/src/components/List/List.types.ts

```diff
@@ -180,7 +231,20 @@ export interface IListProps<T = any> extends React.HTMLAttributes<List<T> | HTML
    * Called when the List will render a page.
    * Override this to control how cells are rendered within a page.
    */
-  onRenderPage?: (pageProps: IPageProps<T>, defaultRender?: IRenderFunction<IPageProps<T>>) => React.ReactNode;
+  onRenderPage?: IRenderFunction<IPageProps<T>>;
+
+  /**
+   * Render override for the element at the root of the `List`.
+   * Use this to apply some final attributes or structure to the content
+   * each time the list is updated with new active pages or items.
+   */
+  onRenderRoot?: IRenderFunction<IListOnRenderRootProps<T>>;
+
+  /**
+   * Render override for the element representing the surface of the `List`.
+   * Use this to alter the structure of the rendered content if necessary on each update.
+   */
+  onRenderSurface?: IRenderFunction<IListOnRenderSurfaceProps<T>>;
```

Comment:
> The expected type comes from property 'onRenderSurface' which is declared here on type 'IntrinsicAttributes & IntrinsicClassAttributes<List<any>> & Pick<Readonly<IListProps<any>> & Readonly<...>, "style" | ... 269 more ... | "ignoreScrollingState"> & Partial<...> & Partial<...>'


There are some more types errors like this: https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=62858&view=logs&j=ca395085-040a-526b-2ce8-bdc85f692774&t=cb33ee24-f492-54d2-727a-ae0a7bb84b2c&l=1234